src/genie/libs/parser/iosxe/tests/ShowApDot115GhzChannel/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,28 @@
+expected_output = {
+    "channel_assignment": {
+        "chan_assn_mode": "AUTO",
+        "chan_upd_int": 12,
+        "anchor_time_hour": 7,
+        "channel_noise": "Enable",
```

Comment:
we usually respect the indentation and create the schema. for this, like below. same applied to other places with indentation.
```
"channel_update_contribution": {
    "noise": "Enable",
    "interference": "Enable",
    (snip)

Reply:
Updated to respect indentation

Reply:
confirmed. thanks for the update.